gm4_scuba_gear/data/scuba_gear/functions/recipe_check.mcfunction

```diff
@@ -0,0 +1,3 @@
+execute if score @s gm4_slot_count matches 6 if block ~ ~ ~ dropper{Items:[{Slot:0b,id:"minecraft:gold_ingot"},{Slot:1b,id:"minecraft:turtle_helmet"},{Slot:2b,id:"minecraft:gold_ingot"},{Slot:3b,id:"minecraft:gold_ingot"},{Slot:4b,id:"minecraft:glass_pane"},{Slot:5b,id:"minecraft:gold_ingot"}]} run data merge block ~ ~ ~ {Items:[{Slot:8,id:"player_head",Count:1b,tag:{gm4_custom_crafters:{multiplier:1},gm4_scuba_gear:{item:"helmet"},display:{Name:"{\"text\":\"SCUBA Helmet\",\"italic\":\"false\"}"},AttributeModifiers:[{AttributeName:"generic.armor",Name:"generic.armor",Amount:2,Operation:0,UUIDLeast:554768,UUIDMost:190244,Slot:"head"},{AttributeName:"generic.attackSpeed",Name:"generic.attackSpeed",Amount:-.5,Operation:0,UUIDLeast:454101,UUIDMost:193195,Slot:"head"}],SkullOwner:{Id:"1e205cc1-461c-4b35-900b-f479f9f250a4",Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NDMxNjc2MjM1NzMsInByb2ZpbGVJZCI6IjVjN2ZiNzhhMmQxMzQ5NTZhNWE1M2EyNGQ5NWY1YjRmIiwicHJvZmlsZU5hbWUiOiJQZWFyc29uSW5tYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzc1OWY1NmU3MDc4NDkzYzEzMDZlZWI2YmQxM2UwYmQ1NTZlYjAxOGUzZGRjMDAzMGE1NzFhYjg4YzRjZGNlNGUifX19"}]}}}}]}
```

Comment:
```suggestion
execute if score @s gm4_slot_count matches 6 if score @s gm4_stack_size mathes 1 if block ~ ~ ~ dropper{Items:[{Slot:0b,id:"minecraft:gold_ingot"},{Slot:1b,id:"minecraft:turtle_helmet"},{Slot:2b,id:"minecraft:gold_ingot"},{Slot:3b,id:"minecraft:gold_ingot"},{Slot:4b,id:"minecraft:glass_pane"},{Slot:5b,id:"minecraft:gold_ingot"}]} run data merge block ~ ~ ~ {Items:[{Slot:8,id:"player_head",Count:1b,tag:{gm4_custom_crafters:{multiplier:1},gm4_scuba_gear:{item:"helmet"},display:{Name:"{\"text\":\"SCUBA Helmet\",\"italic\":\"false\"}"},AttributeModifiers:[{AttributeName:"generic.armor",Name:"generic.armor",Amount:2,Operation:0,UUIDLeast:554768,UUIDMost:190244,Slot:"head"},{AttributeName:"generic.attackSpeed",Name:"generic.attackSpeed",Amount:-.5,Operation:0,UUIDLeast:454101,UUIDMost:193195,Slot:"head"}],SkullOwner:{Id:"1e205cc1-461c-4b35-900b-f479f9f250a4",Properties:{textures:[{Value:"eyJ0aW1lc3RhbXAiOjE1NDMxNjc2MjM1NzMsInByb2ZpbGVJZCI6IjVjN2ZiNzhhMmQxMzQ5NTZhNWE1M2EyNGQ5NWY1YjRmIiwicHJvZmlsZU5hbWUiOiJQZWFyc29uSW5tYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzc1OWY1NmU3MDc4NDkzYzEzMDZlZWI2YmQxM2UwYmQ1NTZlYjAxOGUzZGRjMDAzMGE1NzFhYjg4YzRjZGNlNGUifX19"}]}}}}]}
```
For consistency xD

Reply:
I took this line directly from the heart canisters line; technically since the SCUBA helmet is a skull, there's not a limit to how many you could craft at a time, if you ignored the fact that you can't even stack turtle helmets over 1. Also, there's a typo in that suggestion :P "gm4_stack_size mathes"

Reply:
Well the second like also has that score check but has a water breathing potion in it. It's always best to include those since it's more efficient to check the score first than to later realize the recipe doesn't match.

Reply:
Yeah, I realized that after I commented, and it's been fixed in the "function optimizations" commit